### ISSUE TITLE
Error on line 33

### ISSUE THREAD

**Issue by rschuetzler** [EdricChan03/action-build-deploy-ghpages#2]
I get the following error message when attempting to run the action:

```
/entrypoint.sh: line 33: INPUT_GH_PAGES_COMMIT_MESSAGE: Deploy commit  using EdricChan03action-build-deploy-ghpages in Build GH Pages: syntax error in expression (error token is "commit  using EdricChan03action-build-deploy-ghpages in Build GH Pages")
ERROR: Please use the GH_PAGES_TOKEN to specify the token to use for triggering a build request.
##[error]Docker run failed with exit code 1
```

This is the contents of my `build-pages.yml` file:

```
name: Build GH Pages
on: [push]

jobs:
  build:
    name: Build and deploy GitHub Pages
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@master
      - uses: EdricChan03/action-build-deploy-ghpages@v2.1.0
        with:
          github_token: ${{ secrets.GITHUB_TOKEN }}
          gh_pages_token: ${{ secrets.GH_PAGES_TOKEN }}
```

And I have the GH_PAGES_TOKEN added as a secret on the repo.

**Comment by EdricChan03** [comment]
Thanks for reporting this issue! I've fixed this in the latest version ([`v2.1.1`](../releases/v2.1.1)). You can change `v2.1.0` to use `v2.1.1` instead:

```diff
-uses: EdricChan03/action-build-deploy-ghpages@v2.1.0
+uses: EdricChan03/action-build-deploy-ghpages@v2.1.1
```

**Comment by rschuetzler** [comment]
Now it's this:

```
/entrypoint.sh: line 33: INPUT_GH_PAGES_COMMIT_MESSAGE: Deploy commit  using EdricChan03action-build-deploy-ghpages in Build GH Pages: syntax error in expression (error token is "commit  using EdricChan03action-build-deploy-ghpages in Build GH Pages")
Installing gem bundle...
/entrypoint.sh: line 40: INPUT_REMOTE_REPO: https://x-access-token:***@github.com/rschuetzler/rschuetzler.github.io.git: syntax error in expression (error token is "://x-access-token:***@github.com/rschuetzler/rschuetzler.github.io.git")
/entrypoint.sh: line 46: INPUT_COMMITTER_EMAIL: rschuetzler@users.noreply.github.com: syntax error: invalid arithmetic operator (error token is "@users.noreply.github.com")
##[error]Docker run failed with exit code 1
```

**Comment by EdricChan03** [comment]
I think the issue that you're now facing might be caused by a wrong usage of [parameter substitution](https://www.tldp.org/LDP/abs/html/parameter-substitution.html), which is a wrongdoing on my part. This has since been fixed with [`v2.2.1`](https://github.com/EdricChan03/action-build-deploy-ghpages/releases/tag/v2.2.1). (See #3 for more info)
